packages/cog/src/cog/cutline.ts

```diff
@@ -8,6 +8,12 @@ import { CoveringPercentage, CutlineZoom, ZoomDifferenceForMaxImage } from './co
 import { CogJob, SourceMetadata } from './types';
 
 const PaddingFactor = 1.125;
+type CutlineItem = { path: string; blend: number };
+export const CutlineMap: { [key: string]: CutlineItem } = {
+    sentinel: { path: 's3://basemaps-cog-test/NZCoastCutline.sentinel.geojson', blend: 5 },
+    urban: { path: 's3://basemaps-cog-test/NZCoastCutline.urban.geojson', blend: 20 },
+    rural: { path: 's3://basemaps-cog-test/NZCoastCutline.rural.geojson', blend: 20 },
```

Comment:
I don't think we should have hard coded buckets in here. We could derive the bucket from the input (or output) dir.

Reply:
We may only be able to have one default cutline if we want to avoid making it specific to NZ. Its name should be generic like `default-cutline.geojson.gz`.

Reply:
1. Thanks. Was mentioned above so that's done now. 
2.The sentinel and urban/rural cutlines are distinct. Do you mean a "use this one here if its available" kinda default?

Reply:
Maybe just have one default and for sentinel it has to be specified. That way nothing in the logic is specific to NZ. Of course where to find the default is tricky and may mean it is better to just complain if no cutline is given rather than look for it relative to source or output directories. @blacha what do you think?

Reply:
This seems like something with a wider scope, i.e. building with a deliberate country neutral focus. In the meantime shall we just park the issue?